src/Resources/views/WishlistDetails/_item.html.twig

```diff
@@ -37,11 +41,11 @@
     </div>
 
     <div class="bb-wishlist-item-quantity" {{ sylius_test_html_attribute('wishlist-item-quantity') }}>
-        {{ form_widget(childForm.cartItem.quantity, {
+        {{ form_widget(itemForm.cartItem.cartItem.quantity, {
             'attr': {
                 'min': 0,
                 'data-product-name': product.name,
-                'value': childForm.cartItem.quantity.vars.value is same as("0") ? 1 : childForm.cartItem.quantity.vars.value
+                'value': itemForm.cartItem.cartItem.quantity.vars.value is same as("0") ? 1 : itemForm.cartItem.cartItem.quantity.vars.value
```

Comment:
Why same as 0?

Reply:
It's done by Patryk, smae as 0 because it is default value of quantity, and if so it is changed to 1